README.md

```diff
@@ -1,5 +1,21 @@
 ![](docs/images/logo.png)
 
+<p align="center">
+  <a href="https://github.com/mxpv/shim-rs/actions/workflows/ci.yml">
+    <img alt="GitHub Workflow Status" src="https://github.com/mxpv/shim-rs/actions/workflows/ci.yml/badge.svg?branch=main">
+  </a>
```

Comment:
Should fetch CI status from kuasar-io/kuasar actions

```suggestion
  <a href="https://github.com/kuasar-io/kuasar/actions/workflows/ci.yml">
    <img alt="GitHub Workflow Status" src="https://github.com/kuasar-io/kuasar/actions/workflows/ci.yml/badge.svg?branch=main">
  </a>
```

Reply:
@kevin-wangzefeng Thanks for the suggestion, my carelessness made this mistake. Feel sorry.